src/app/public/styles/ag-grid-styles.scss

```diff
@@ -421,6 +421,26 @@ ag-grid-angular.sky-ag-grid-editable {
       }
     }
   }
+
+  .ag-popup {
```

Comment:
Why are we adding new styles in this PR?

Reply:
There were issues with the dark theme that I noticed when reviewing the e2e results.